docs/guides/creatingasource.md

```diff
@@ -12,34 +12,60 @@ We will create:
 - [x] The `sockeye` target which serves as an event display.
 - [x] The `AWSSQSSource` which consumes events from an AWS SQS queue.
 
-## Event display
+!!! tip "Kubernetes namespace"
+    All objects mentioned in this guide must be created inside the same Kubernetes namespace.
+
+## Sockeye CloudEvents viewer display
 
 First of all, we need to have a tool to see the events that come from our source.
 
-Create a `sockeye` service by saving the following YAML manifest in a file called `sockeye.yaml` and applying it to your Kubernetes cluster:
+Create a `sockeye` deployment and service by saving the following YAML manifest in a file called `sockeye.yaml` and applying it to your Kubernetes cluster:
 
 ```yaml
-apiVersion: serving.knative.dev/v1
+apiVersion: v1
 kind: Service
 metadata:
   name: sockeye
 spec:
+  selector:
+    app.kubernetes.io/name: sockeye
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 8080
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sockeye
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: sockeye
   template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: sockeye
     spec:
       containers:
-        - image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
+      - name: sockeye
+        image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
 ```
 
-```
+```console
 kubectl apply -f sockeye.yaml
```

Comment:
The $ sign is missing here, but other commands have it. I don't have a preference, it should just be consistent.